test/index.js

```diff
@@ -110,8 +118,11 @@ describe('simple-keyring', function () {
     const expectedResult =
       '0x28fcb6768e5110144a55b2e6ce9d1ea5a58103033632d272d2b5cf506906f7941a00b539383fd872109633d8c71c404e13dba87bc84166ee31b0e36061a69e161c';
 
-    it('passes the dennis test', async function () {
+    beforeEach(async function () {
```

Comment:
Do you mind putting this in each test instead of using a `beforeEach`? I think it would be easier to read and understand.

Reply:
changed